### ISSUE TITLE
Second QCoreApplication apparently can lead to test failures

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/OpenMS#6329]
https://github.com/OpenMS/OpenMS/blame/9a45bd1e7dd9be3f9bb073d04c22c5ced79688e5/src/topp/IDFileConverter.cpp#L826

From Discord:

```
Command: "C:/Users/OpenMS_Windows64/bin/Debug/IDFileConverter.exe" "-in" "IDMerger_6_output1.oms" "-out" "IDMerger_6_output2.tmp" "-out_type" "idXML"
Directory: C:/Users/OpenMS_Windows64/src/tests/topp
"TOPP_IDMerger_6_out1" start time: Sep 06 20:17 Eastern Daylight Time
Output:
----------------------------------------------------------
ASSERT failure in QCoreApplication: "there should be only one application object", file kernel\qcoreapplication.cpp, line 772
<end of output>
```

of course related to #5781

**Comment by jpfeuffer** [comment]
Fixed in #6306

**Comment by jpfeuffer** [comment]
@cbielow can you double check that it still works for you?

I don't think using a QCoreApplication was a correct solution. In case it still fails we might need to copy the Qt plugin folder with CMake during build time already. But actually at build time everything should still be running from the original Qt directory structure.
And @poshul is currently working on shipping plugins for the installers.
